apps/wallet/src/shared/utils/bip39.test.ts

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) 2022, Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { describe, it, expect } from 'vitest';
+
+import { normalizeMnemonics, validateMnemonics } from './bip39';
+
+describe('mnemonics', () => {
+    it('normalize mnemonics', () => {
+        expect(normalizeMnemonics(' Almost a Seed    Phrase')).toEqual(
+            'almost a seed phrase'
+        );
+    });
+
+    it('validate mnemonics', () => {
+        // Mnemonics length too short
+        expect(validateMnemonics('result')).toBe(false);
+
+        // Mnemonics length too long
```

Comment:
This comment should be "wrong length" not "too long", but we could try a test with too long as well (ie 25+ words)